.github/workflows/deploy-i18n.yml

```diff
@@ -134,10 +134,54 @@ jobs:
           docker push registry.digitalocean.com/${{ secrets.DOCR_NAME }}/${{ matrix.site_tlds }}/news-${{ matrix.languages }}:$tagname
           docker push registry.digitalocean.com/${{ secrets.DOCR_NAME }}/${{ matrix.site_tlds }}/news-${{ matrix.languages }}:latest
 
+      - name: Clean old DO images
+        uses: actions/github-script
+        with:
+          script: |
+            const { exec } = require('child_process');
+            const { promisify } = require('util');
+            
+            const asyncExec = promisify(exec);
+            const { stdout: rawTags } = await asyncExec("doctl registry repository list-tags org/news-${{ matrix.languages }} --output=json --context fcc");
+            const tags = JSON.parse(rawTags);
+            const toDelete = [];
+            for (const tag of tags) {
+                // if tag.updated_at is more than two days ago
+                if (new Date(tag.updated_at).getTime() < Date.now() - (1000 * 60 * 60 * 24 * 2)) {
+                  toDelete.push(tag.tag);
+                }
+            }
+            if (!toDelete.length) {
+                console.log("No outdated tags found.");
+                return;
+            }
+            console.log(`Found ${toDelete.length} outdated tags.`);
+            for (const tagName of toDelete) {
+                await asyncExec(`doctl registry repository delete-tag org/news-${{ matrix.languages }} ${tagName} --context fcc`)
+            }
+
       - name: Clean up Self-hosted setup
         uses: colpal/actions-clean@36e6ca1abd35efe61cb60f912bd7837f67887c8a # tag=v1
         if: ${{ always() }} # To ensure this step runs even when earlier steps fail
 
+  clean:
+    name: Clean up DO images
```

Comment:
Because garbage collection runs at the registry level, this needs to run after all of the tags have been cleaned (rather than running once on each lang).

Additionally, because it sets the registry to read-only mode until the GC is complete, it needs to run after all of the new images are pushed.

Reply:
Is it possible to move the step `Clean old DO images` into this job?